styles.css

```diff
@@ -0,0 +1,366 @@
+
+
+h1 {
+  font-family: 'Bitter', serif;
+  color: #093C60;
+}
+
+h2 {
+  font-family: 'Indie Flower', cursive;
+  color: #81B742;
+}
+
+h3 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+}
+
+h3.footer-h3 {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+h4 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+  line-height: 0.5;
+}
+
+p {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+p.small {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+  font-size: 9pt;
+}
+
+p.light{
+  color: #ECECEC;
+  text-align: right;
+}
+
+p.footer-p {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+a {
+  color: inherit;
+  font-family: inherit;
+  text-decoration: none;
+  margin: -10px -10px -10px -10px;
```

Comment:
You can shorthand these (and the padding below) if all four sides get the same values

```
margin: -10px;
```

I'm not a fan of using negative margin unless it's really needed, which, I'd argue this isn't a place where it's needed.